vtexdocs/RapiDoc
# Feat/breadcrumbs tooltip

#### What is the purpose of this pull request?

Add the tooltip component to the breadcrumbs. Further information can be found [here](https://www.notion.so/vtexhandbook/Adicionar-tooltip-em-breadcrumbs-do-RapiDoc-bf9cb79de822451e82c57be5972d845c).

#### What problem is this solving?

Long strings had the number of characters limited and could not be read in full.

#### How should this be manually tested?

Open the preview, open different api reference pages and check if a tooltip appears in the breadcrumbs that contain ellipsis.

#### Screenshots or example usage

![Captura de tela de 2023-01-03 11-11-36](https://user-images.githubusercontent.com/62757720/210387203-b37f876f-f72d-43fd-b328-db2d4d6d9ad9.png)
![Captura de tela de 2023-01-03 11-12-14](https://user-images.githubusercontent.com/62757720/210387166-543cd940-f3ec-4b4d-ac40-a00a25305858.png)

#### Types of changes
